test/built-ins/TypedArrays/buffer-arg-proto-from-ctor-realm-sab.js

```diff
@@ -24,8 +24,7 @@ info: |
        b. Let proto be realm's intrinsic object named intrinsicDefaultProto.
     5. Return proto.
 includes: [testTypedArray.js]
-features: [SharedArrayBuffer]
-features: [Reflect]
+features: [SharedArrayBuffer,Reflect]
```

Comment:
thanks for fixing it!

For consistency, can you just add a space after the comma character?